WcaOnRails/spec/controllers/committee_positions_controller_spec.rb

```diff
@@ -0,0 +1,187 @@
+require 'rails_helper'
+
+RSpec.describe CommitteePositionsController, type: :controller do
+  let!(:committee) { FactoryGirl.create(:committee) }
+  let!(:committee_position) { FactoryGirl.create(:committee_position) }
+  let!(:committee_position_no_members) { FactoryGirl.create(:committee_position, name: "Committee position with no members") }
+  let!(:team_member) { FactoryGirl.create(:team_member) }
+
+  describe "GET #show" do
+    it 'produces UrlGenerationError as there is no show page' do
+      expect {
+        get :show, {committee_id: committee_position.committee.slug}
+      }.to raise_error(ActionController::UrlGenerationError)
+    end
+  end
```

Comment:
Sorry, but I don't see how this could be valuable. For me it's almost like testing whether `GET #foo` raises the error. But maybe @larspetrus have another opinion =) ?